src/components/DownloadReleases/index.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+
 import { ReleaseData } from '../../hooks/useReleaseHistory';
-import DownloadTable from './DownloadTable';
 import hexagonFilled from '../../images/icons/hexagon-filled.svg';
+import DownloadTable from './DownloadTable';
```

Comment:
I think you meant to move this either above or below the hexagon svg's

Reply:
Good catch 😄 Now imports are grouped.